recipes-robotframework/ref-unit-setup/ref-unit-setup/ref_can.network

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=can*
+
+[CAN]
+BitRate=500K
+RestartSec=100ms
```

Comment:
* we can have a longer RestartSec here to make it easier to detect errorframes
* we can enable canfd here as well with 
```
[CAN]
DataBitRate=4000000
FDMode=True
```
* https://pengutronix.de/en/blog/2022-02-04-initializing-can-interfaces-with-systemd-networkd.html